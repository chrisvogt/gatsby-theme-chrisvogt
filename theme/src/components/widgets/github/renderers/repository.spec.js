import React from 'react'
import renderer from 'react-test-renderer'
import Repository from './repository'

describe('GitHub Repository Renderer', () => {
  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Repository
          description='GatsbyJS blog theme with built-in social widgets for creatives and developers.'
          nameWithOwner='themeuser/gatsby-theme-chronogrove'
          openGraphImageUrl='https://avatars0.githubusercontent.com/u/1934719?s=400&v=4'
          updatedAt='2020-03-26T12:11:58Z'
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
