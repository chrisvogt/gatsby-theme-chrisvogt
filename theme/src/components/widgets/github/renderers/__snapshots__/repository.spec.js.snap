// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHub Repository Renderer matches the snapshot 1`] = `
<div
  className="css-gfj6dx-Repository"
>
  <h4
    className="css-11khky8-Repository"
  >
    chrisvogt/gatsby-theme-chrisvogt
  </h4>
  <span
    className="css-rpa521-Repository"
  >
    GatsbyJS blog theme with built-in social widgets for creatives and developers.
  </span>
  <div
    className="css-1tkdyol-CardFooter"
  >
    <span>
      Last updated 
      4 years ago
    </span>
    <span>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-up-right-from-square "
        data-icon="up-right-from-square"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={{}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M352 0c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9L370.7 96 201.4 265.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L416 141.3l41.4 41.4c9.2 9.2 22.9 11.9 34.9 6.9s19.8-16.6 19.8-29.6V32c0-17.7-14.3-32-32-32H352zM80 32C35.8 32 0 67.8 0 112V432c0 44.2 35.8 80 80 80H400c44.2 0 80-35.8 80-80V320c0-17.7-14.3-32-32-32s-32 14.3-32 32V432c0 8.8-7.2 16-16 16H80c-8.8 0-16-7.2-16-16V112c0-8.8 7.2-16 16-16H192c17.7 0 32-14.3 32-32s-14.3-32-32-32H80z"
          fill="currentColor"
          style={{}}
        />
      </svg>
    </span>
  </div>
</div>
`;
